### DISCUSSION TITLE
ACE Bodybag not working

### DISCUSSION THREAD

**Discussion by XDSenDX** [Vdauphin/HeartsAndMinds#1552] [Question and answer] [answered]
Unable to retrieve tickets even the body bag is right next to the box.

**Comment by Vdauphin** [comment, marked as answer]
Hello,

Just move the body bag on top of the logistic point helipad and interact with the logistic red box
Also please read the in game documentation: http://vdauphin.github.io/HeartsAndMinds/InGame-documentation#get-new-tickets

Cheers
